### PR TITLE
uhub: re-enable sqlite plugin / fix aarch64 build

### PR DESCRIPTION
**Pull request by makefu**
###### Motivation for this change

This PR re-enables the sqlite authentication plugin for uhub. It was disabled because of a failed build, but this seems to have been solved.
Additionally it fixes the build of uhub for aarch64

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by makefu** [comment]
@GrahamcOfBorg build uhub

**Comment by Ekleog** [comment]
@makefu Any reference to an upstream issue where the patch you're adding is discussed? This would be helpful to be able to eventually remove it, I think :)

**Comment by makefu** [comment]
@Ekleog i've added a reference to the debian bug (open since 2014)

**Comment by Ekleog** [comment]
I've just opened https://github.com/janvidar/uhub/issues/46 upstream based on the bug you linked (maybe the author never saw the debian bug), you may want to link to it instead? :)

**Comment by makefu** [comment]
i've removed the patch file and instead pull it from the github PR i've opened

**Comment by Ekleog** [comment]
@makefu Looks like a problem with the patch hash now :/ (don't know if you read ofborg's emails)

**Comment by makefu** [comment]
@Ekleog weird thing, let me try again

**Comment by Ekleog** [comment]
@makefu I don't know, sorry, it's sent through the same mechanism as all github comments theoretically :/ maybe in your spam folder?

Anyway, I think that's ready to be merged :)

**Comment by makefu** [comment]
ah you mean *these* mails :D yes, i get them but i was busy doing daily work

**Comment by makefu** [comment]
anything holding this PR to be merged?
